tree/dataframe/inc/ROOT/RDF/RCustomColumn.hxx

```diff
@@ -95,9 +95,9 @@ class RCustomColumn final : public RCustomColumnBase {
    }
 
 public:
-   RCustomColumn(RLoopManager *lm, std::string_view name, F &&expression, const ColumnNames_t &columns,
+   RCustomColumn(RLoopManager *lm, std::string_view name, F expression, const ColumnNames_t &columns,
```

Comment:
Is it the intent to 'force' a copy of the expression upon a call to this function even in case where the user argument was a temporary that could have been moved into place?

Reply:
so, previous code was completely wrong, because it only worked if the argument passed to the constructor was an rvalue. the new version always takes a copy, it's true, but i'd rather have that than complicating the class with a template constructor...this code is far away from any hot path.